src/Plug/Math/Vec2d.h

```diff
@@ -0,0 +1,168 @@
+#ifndef _VEC2D_H_
+#define _VEC2D_H_
```

Comment:
I suggest making all include guards the same - __PLUG_<FILENAME>_H for clarity and to avoid possible conflicts with include guards with the same name in our own projects